src/utils.js

```diff
@@ -191,6 +191,20 @@ export function buildNodeTree(element) {
     return tree
 }
 
+function findNode(children) {
+    while (children.length) {
+        const child = children.shift()
+
+        if (child.node) {
+            return child.node
+        }
+
+        if (child.children && Array.isArray(child.children)) {
+            children.push(...child.children)
```

Comment:
this findNode() method is destructive. It modifies the `children` parameter that's passed on to it. You could instead, just recursively call findNode() on the children node until a node is found

Reply:
findNode method using a copy of children, so original child not affected by the method

https://github.com/baruchvlz/resq/pull/37/files#diff-2b4ca49d4bb0a774c4d4c1672d7aa781R227